website/content/en/pre-docs/getting-started/_index.md

```diff
@@ -194,6 +196,10 @@ spec:
     - key: karpenter.sh/capacity-type
       operator: In
       values: ["spot"]
+  limits:
+    resources:
+      cpu: 1000
+      memory: 1000Gi
```

Comment:
Could you remove this line? I know it was mine, but I think just CPU defined limits are good enough.

Reply:
👍